src/Security/SecurityConfigUpdater.php

```diff
@@ -185,34 +183,33 @@ private function updateProviders(UserClassConfiguration $userConfig, string $use
         $this->manipulator->setData($newData);
     }
 
-    private function updatePasswordHashers(UserClassConfiguration $userConfig, string $userClass, string $keyName = 'password_hashers'): void
+    private function updatePasswordHashers(UserClassConfiguration $userConfig, string $userClass): void
     {
         $newData = $this->manipulator->getData();
-        if ('password_hashers' === $keyName && isset($newData['security']['encoders'])) {
-            // fallback to "encoders" if the user already defined encoder config
-            $this->updatePasswordHashers($userConfig, $userClass, 'encoders');
 
```

Comment:
Re for situations where a user still has `security.encoders` set - we could do any of:
A) Throw an exception  e.g. "MakerBundle does not support password encoders.."
B) Remove the existing `security.encoders` and continue with setting `security.password_hashers`

The problem with the latter is we dont have a way to tell the user that we're removing the existing encoders.. I suppose we _could_ start linking `make:user`'s IO to `SecurityConfigUpdater` or something to make this "async" but that just feels like a headache w/ not alot of gain. 

If we just throw an exception, what action do we tell the user to take to correct the problem?

Reply:
If `encoders` is set, let's throw a clear exception that explains to change to `password_hashers`.

Reply:
sounds like a plan